pkg/controller/status/status.go

```diff
@@ -879,3 +879,58 @@ func hasPendingCSRUsingCertV1beta1(ctx context.Context, cli client.Client, label
 	}
 	return false, nil
 }
+
+func UpdateStatusCondition(statuscondition []metav1.Condition, conditions []operator.TigeraStatusCondition, generation int64) []metav1.Condition {
```

Comment:
Please add a comment to explain how to use this function.

Reply:
Done